src/main/resources/templates/my-forums.html

```diff
@@ -1,85 +1,91 @@
+            }
+
+            main > div:first-child > h2 {
+                position: absolute;
+                bottom: 0;
+                margin: 1rem;
+                color: white;
+                text-shadow: 0 0 3px black;
+            }
 
-      main > div:first-child img {
-        border-radius: 1rem;
-        border: 1px solid black;
+      .mops-row {
+        transition-duration: 0.3s;
       }
 
-      main > div:first-child > h2 {
-        position: absolute;
-        bottom: 0;
-        margin: 1rem;
-        color: white;
-        text-shadow: 0 0 3px black;
+      .mops-row>*:hover {
+        cursor: pointer;
       }
 
-      @media (min-width: 992px) {
-        main > div:first-child > h2 {
-          font-size: 2.4rem;
-        }
+      .mops-row:hover {
+        background: #a9d0f5;
+        transition-duration: 0.3s;
       }
 
-      @media (max-width: 576px) {
+      @media (min-width: 992px) {
         main > div:first-child > h2 {
-          font-size: 1.0rem;
+          font-size: 2.4rem;
         }
       }
-    </style>
-  </th:block>
+            @media (max-width: 576px) {
+                main > div:first-child > h2 {
+                    font-size: 1.0rem;
+                }
+            }
+        </style>
+    </th:block>
 </head>
 <body>
 <header>
-  <nav class="navigation navigation-secondary" is="mops-navigation" th:fragment="navigation">
-    <ul>
-      <li class="active">
-        <a href="#welcome" th:href="@{./}">Dashboard</a>
-      </li>
-      <li class="">
-        <a href="#welcome" th:href="@{./profile}">Profil</a>
-      </li>
-      <li class="">
-        <a href="#welcome" th:href="@{./blank}">Blank</a>
-      </li>
-      <li class="">
-        <a href="#welcome" th:href="@{./sidenav}">Sidenav</a>
-      </li>
-      <li class="">
-        <a href="#welcome" th:href="@{./#welcome}">Propra 2</a>
-      </li>
-
-    </ul>
-  </nav>
+    <nav class="navigation navigation-secondary" is="mops-navigation" th:fragment="navigation">
+        <ul>
+          <li class="">
+            <a href="" th:href="@{./}">Dashboard</a>
+          </li>
+          <li class="active">
+            <a href="" th:href="@{./my-forums}">Meine Foren</a>
+          </li>
+          <li class="">
+            <a href="" th:href="@{./profile}">Profil</a>
+          </li>
+        </ul>
+    </nav>
 </header>
 <main th:fragment="bodycontent">
-
   <h1>Meine Foren:</h1>
-  <th:block th:each="forum : ${forums}">
-  <div class="row">
-      <div class="card">
-        <div class="card-header">
-          Propra2
-        </div>
-        <div class="card-body">
-          <h5 class="card-title">offizielles Forum</h5>
-          <a href="#" class="btn btn-primary">einsehen</a>
-        </div>
-      </div>
-  </div>
+  <table class="table table-light">
+    <thead class="thead-light">
+    <tr>
+      <th>Name</th>
+      <th>Letzte Änderung</th>
+    </tr>
+    </thead>
+    <tbody>
+    <div th:each="forum : ${forums}">
```

Comment:
using th:object here? can you omit to string?

Reply:
I generally use th:each to iterate through collections. What is your argument for th:object?